libgnucash/engine/Account.h

```diff
@@ -924,6 +924,12 @@ Account *gnc_account_lookup_by_full_name (const Account *any_account,
 Account *gnc_account_lookup_by_code (const Account *parent,
                                      const char *code);
 
+/** The gnc_account_lookup_by_by_opening_balance () subroutine
+ *  works like gnc_account_lookup_by_name, but checks for the
+ *  opening balance flag set.
+ */
```

Comment:
Doxygen will fill in the function name so lead off with an action verb. Don't refer to other functions in the description, they might change making your description inaccurate. Describe each parameter beginning with `@param` and what the function returns beginning with @return:
```C
/** Find the opening balance account for the currency.
 *
 *  @param parent The account of which the sought-for account is a descendant.
 *  @param commodity The commodity in which the account should be denominated
 *  @return The descendant account of EQUITY_TYPE_OPENING_BALANCE or NULL if one doesn't exist.
 */
```